css/properties/position-area.json

```diff
@@ -58,8 +57,7 @@
               "chrome_android": "mirror",
               "edge": "mirror",
               "firefox": {
-                "version_added": false,
-                "impl_url": "https://bugzil.la/1838746"
+                "version_added": "131"
```

Comment:
I would consider all values as not supported, as they have no effect.

Reply:
But the `position-area` property does parse so technically is supported.